Runtime/AssemblyAttributes.cs

```diff
@@ -3,3 +3,4 @@
 [assembly: InternalsVisibleTo("Innoactive.CreatorEditor")]
 [assembly: InternalsVisibleTo("Innoactive.Creator.Core.Tests")]
 [assembly: InternalsVisibleTo("Innoactive.Creator.Core.Tests.Editmode")]
+[assembly: InternalsVisibleTo("Innoactive.Creator.DesktopInteraction")]
```

Comment:
Just out of curiosity? What does the desktop mode use from the core?

Reply:
PropertyReflectionHelper.cs 😇